src/sysupdate/sysupdate.c

```diff
@@ -794,7 +850,9 @@ static int context_apply(
         log_info("Selected update '%s' for install.", us->version);
 
         (void) sd_notifyf(false,
-                          "STATUS=Making room for '%s'.", us->version);
+                          "READY=1\n" /* Tell sysupdated that we've started working */
+                          "X_UPDATE_VERSION=%s\n"
```

Comment:
Looks like you've missed some of the review comments Lennart left, such as this one. Please double-check you've actually resolved all the review comments

I usually mark the review comments as resolved on GitHub as I solve them, to avoid mistakes like this :shrug: . Having everything neatly solved in GitHub will make the review go easier too